src/actions/actions/services/dispatchers/UpdateServiceNotifications.js

```diff
@@ -0,0 +1,26 @@
+import {
+    ERROR_SERVICE_ACCOUNT,
+    SET_SERVICE_ACCOUNT
+  } from "../../../../utils/constants/storeType";
+  import { VERUSID } from "../../../../utils/constants/intervalConstants";
+  import { updateServiceDataValue } from "./UpdateServiceDataValue";
+
+  import { checkVerusIdNotificationsForUpdates } from "./verusid/verusid";
+  
+  const fetchChannels = () => {
+    return {
+      [VERUSID]: (channelStore) =>
+        checkVerusIdNotificationsForUpdates(channelStore),
+    };
+  };
+  
+  export const updateServiceNotifications = (state, dispatch, channels) =>
```

Comment:
Having an updateServiceNotifications function is good, but as mentioned before, it should follow the schema of check service data -> find out what needs to be notified -> push general notifications (not service specific) if anything should be notified.